koku/reporting/provider/ocp/models.py

```diff
@@ -299,6 +299,19 @@ class Meta:
     label = ArrayField(models.TextField())
 
 
+class OpenshiftCostCategoryNamespace(models.Model):
+    """Namespaces to bucket to category."""
+
+    class Meta:
+        """Meta for cost category namespaces."""
+
+        db_table = "reporting_ocp_cost_category_namespace"
+
+    namespace = models.TextField(unique=True)
```

Comment:
Just thinking about this a little bit, this field doesn't _really_ represent "namespace". Like, `openshift-%` is not a namespace. I wonder if we should name this something like "namespace_pattern"?